From bd709251a971ce7855857ba230123605cfa710e7 Mon Sep 17 00:00:00 2001
From: Vladimir Smirnov <civil@gentoo.org>
Date: Mon, 19 May 2014 19:54:51 +0400
Subject: [PATCH] Cleanup PATH and shrpenv

cut the crap of inventing paths, or adding search paths that we don't
use

Patch-Name: gentoo/cleanup-paths.diff
Original-By: Torsten Veller <tove@gentoo.org>
---
 Configure | 79 ---------------------------------------------------------------
 1 file changed, 79 deletions(-)

diff --git a/Configure b/Configure
index b221390..63b8848 100755
--- a/Configure
+++ b/Configure
@@ -107,28 +107,6 @@
 	fi
 fi
 
-: Proper PATH setting
-paths='/bin /usr/bin /usr/local/bin /usr/ucb /usr/local /usr/lbin'
-paths="$paths /opt/bin /opt/local/bin /opt/local /opt/lbin"
-paths="$paths /usr/5bin /etc /usr/gnu/bin /usr/new /usr/new/bin /usr/nbin"
-paths="$paths /opt/gnu/bin /opt/new /opt/new/bin /opt/nbin"
-paths="$paths /sys5.3/bin /sys5.3/usr/bin /bsd4.3/bin /bsd4.3/usr/ucb"
-paths="$paths /bsd4.3/usr/bin /usr/bsd /bsd43/bin /opt/ansic/bin /usr/ccs/bin"
-paths="$paths /etc /usr/lib /usr/ucblib /lib /usr/ccs/lib"
-paths="$paths /sbin /usr/sbin /usr/libexec"
-paths="$paths /system/gnu_library/bin"
-
-for p in $paths
-do
-	case "$p_$PATH$p_" in
-	*$p_$p$p_*) ;;
-	*) test -d $p && PATH=$PATH$p_$p ;;
-	esac
-done
-
-PATH=.$p_$PATH
-export PATH
-
 : shall we be using ksh?
 inksh=''
 needksh=''
